InactiveIssue/GetInactiveIssue.py

```diff
@@ -0,0 +1,71 @@
+from github import Github
+#from inactiveIssue.github_token import GITHUB_TOKEN
+from dateutil.parser import parse
+import time
+from datetime import date
+from django.http import HttpResponse
+from gci.config import get_api_key
+import requests,json
+
+try:
+    GH_TOKEN = get_api_key('GH')
+except:
+    print("Unable to get the token")
+    pass
```

Comment:
this does nothing.

but also get rid of code at the module level.  that is always a bad idea.